styles.css

```diff
@@ -201,16 +202,178 @@ body {
     font-size: 16px;
     line-height: 1.75;
     color: var(--color-light-gray);
-    background-color: var(--color-dark);
+    background-color: var(--color-dark);    
 }
 
+body {
+    background-color: #000000; 
+}
+
+a,
+button,
+input,
+textarea,
+svg * {
+	transition-duration: var(--transition-duration);
+}
+
+a {
+    color: var(--color-light);
+}
+
+a:hover {
+	color: var(--color-light-gray);
+}
 container {
     max-width: calc(var(container-width) + var(container-padding-x) * 2);
     margin-inline: auto;
     padding-inline: var(--container-padding-x);
 }
 
+.visualy-hidden {
+    position: absolute !important;
+    width: 1px !important;
+    height: 1px !important;
+    margin: -1px !important;
+    border: 0 !important;
+    padding: 0 !important;
+    white-space: nowrap !important;
+    clip-path: inset(100%) !important;
+    clip: rect(0 0 0 0) !important;
+    overflow: hidden !important;
+}
+
 h1, h2, h3, h4, h5, h6 {
     font-weight: 700;
     text-transform: uppercase;
+}
+
+.button {
+		display: inline-flex;
+		align-items: center;
+		height: 38px;
+		padding-inline: 26px;
+		font-size: 12px;
+		font-weight: 700;
+		line-height: 2;
+		letter-spacing: 1.2px;
+		text-transform: uppercase;
+		white-space: nowrap;
+		background-color: var(--color-light);
+		border: none;
+		border-radius: var(--border-radius); 
+}
+
+.button:hover {
+		background-color: var(--color-light-gray);
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    column-gap: 20px;
+		padding-inline: 32px;
+    background-color: var(--color-dark-alternate);
+}
+
+.header-logo {
+	flex-shrink: 0;
+}
+
+.header-menu-list {
+    display: flex;
+		flex-wrap: wrap;
```

Comment:
Why have you merged this PR without review?
And why do you need this huge spaces `                      flex-wrap: wrap;`?

Reply:
2 or 4 spaces is enough: `    flex-wrap: wrap;`